meta-lmp-base/recipes-samples/images/lmp-image-common.inc

```diff
@@ -37,6 +37,7 @@ CORE_IMAGE_BASE_INSTALL += " \
     nss-altfiles \
     pam-plugin-mkhomedir \
     resize-helper \
+    luks-reencryption \
```

Comment:
Have a conditional here, since we only want this to be installed when luks is required by the distro.

E.g.
`${@bb.utils.contains('DISTRO_FEATURES', 'luks', 'luks-reencryption', '', d)}`